libethereum/BlockChain.cpp

```diff
@@ -347,7 +347,7 @@ pair<h256s, h256> BlockChain::attemptImport(bytes const& _block, OverlayDB const
 	}
 }
 
-pair<h256s, h256> BlockChain::import(bytes const& _block, OverlayDB const& _db, Aversion _force)
+pair<h256s, h256> BlockChain::import(bytes const& _block, OverlayDB const& _db, Aversion _force, bool _checkNonce)
```

Comment:
latest update to coding standards: avoid bool params; favour enums (just like Aversion, next to it :-) )

consider something like:
```
struct ImportRequirements {
using value = unsigned;
enum { ValidNonce = 1, DontHave = 2 };
};
pair<h256s, h256> BlockChain::import(bytes const& _block, OverlayDB const& _db, ImportRequirements::value _f = ImportRequirements::ValidNonce | ImportRequirements::DontHave);
```